### PR TITLE
Disable gssapi for linux builds

### PR DESCRIPTION
**Pull request by rnpridgeon**

### CODE REVIEW

File: tools/bootstrap-librdkafka.sh [hunk 1]
```diff
@@ -34,7 +34,12 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 
 ./configure --clean
 make clean
-./configure --install-deps --source-deps-only --prefix="$INSTALLDIR"
+
+if [[ "$OSTYPE" == "linux"* ]]; then
```
Comment (edenhill) [inline tools/bootstrap-librdkafka.sh]: nit: the quotes are not needed with double-bracket statements.

File: tools/bootstrap-librdkafka.sh [hunk 2]
```diff
@@ -34,7 +34,12 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 
 ./configure --clean
 make clean
-./configure --install-deps --source-deps-only --prefix="$INSTALLDIR"
+
+if [[ uname -s  != "Darwin" ]]; then
+  MKL_OPTS="--disable-gssapi"
```
[the remark this answers is not in the document]
Comment (edenhill) [reply tools/bootstrap-librdkafka.sh]: Please change the name of this variable to not interfere with mklove. E.g., EXTRA_OPTS

[the remark this answers is not in the document]
Comment (rnpridgeon) [reply tools/bootstrap-librdkafka.sh]: 👍

File: tools/bootstrap-librdkafka.sh [hunk 3]
```diff
@@ -34,7 +34,12 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 
 ./configure --clean
 make clean
-./configure --install-deps --source-deps-only --prefix="$INSTALLDIR"
+
+if [[ uname -s  != "Darwin" ]]; then
```
[the remark this answers is not in the document]
Comment (rnpridgeon) [reply tools/bootstrap-librdkafka.sh]: $OSTYPE appears to be fairly portable (tried centos, ubuntu, osx)

File: tools/bootstrap-librdkafka.sh (hunk 1 above)
Comment (rnpridgeon) [reply tools/bootstrap-librdkafka.sh]: ack

File: tools/bootstrap-librdkafka.sh [hunk 4]
```diff
@@ -35,11 +35,11 @@ curl -q -L "https://github.com/edenhill/librdkafka/archive/${VERSION}.tar.gz" |
 ./configure --clean
 make clean
 
-if [[ "$OSTYPE" == "linux"* ]]; then
-  MKL_OPTS="--disable-gssapi"  
+if [[ $OSTYPE == "linux"* ]]; then
```
Comment (edenhill) [inline tools/bootstrap-librdkafka.sh]: micro nit: you dont need to quote on the right hand side either as long as it is just a single word. (but dont change it, just pointing it out)

**Pull request merged**